db/migrate/20221014124812_create_contents.rb

```diff
@@ -0,0 +1,9 @@
+class CreateContents < ActiveRecord::Migration[6.1]
```

Comment:
I'm a bit unsure about the current title of the table. Perhaps `curriculum_content` is more descriptive? Alternatively `html_content` or similar, but then it's tied to an implentation detail rather than its role.